themes/opentermsarchive/layouts/partials/related-datasets.html

```diff
@@ -2,6 +2,9 @@ <h2>{{ .Params.related_datasets.title }}</h2>
 <div>{{ .Params.related_datasets.introduction | markdownify }}</div>
 <ul>
   {{ range site.Data.related_datasets }}
-    <li>name : {{ .name }}<br/>website : {{ .website }}<br />description : {{ index .description site.Language.Lang }}</li>
+  <li>
+    name : {{ .name }}<br>
```

Comment:
In English, there is no space after `:`. Maybe use a definition list

Reply:
No space _before_ 🙂

Reply:
Yes, you're right, that's what I meant :D

Reply:
I'll fix that when I'll apply the styles.